nautapy/cli.py

```diff
@@ -238,8 +238,20 @@ def run_connected(args):
     user, password = _get_credentials(args)
     client = NautaClient(user, password)
 
-    with client.login():
-        os.system(" ".join(args.cmd))
+    if not NautaProtocol.is_connected():
+        with client.login():
+            os.system("".join(args.cmd))
+    else:
```

Comment:
Falta checkear el flag `--reuse-connection`

```suggestion
    elif args.reuse_connection:
```